app.js

```diff
@@ -0,0 +1,295 @@
+import './style.css';
+import kbdkeys from './keys';
+
+const states = {
+  firedKey: null,
+  ControlLeft: false,
+  AltLeft: false,
+  ShiftLeft: false,
+  position: null,
+  lang: 'en',
+};
+
+const createEl = (tag, cls, addTo, tagvalue) => {
+  const el = document.createElement(tag);
+  if (cls != null) {
+    el.setAttribute('class', cls);
+  }
+  if (tagvalue != null) {
+    el.innerText = tagvalue;
+  }
+
+  addTo.append(el);
+  return el;
+};
+
+const changeLang = (kbdKeysInst) => {
+  const { lang, ShiftLeft } = states;
+  if (lang === 'en' && ShiftLeft === false) {
+    kbdKeysInst.forEach((keyGroup) => {
```

Comment:
The logic inside 'if' looks very similar for all cases, I believe, it's possible to unite it